src/__tests__/partial-fulfill.spec.ts

```diff
@@ -147,6 +147,79 @@ describeWithFixture(
           expect(fulfillStandardOrderSpy).calledOnce;
         });
 
+        it("ERC1155 <=> ETH doesn't fail due to rounding error", async () => {
+          const { seaport, testErc1155 } = fixture;
+
+          // broke out key params to make testing different values easier:
+          const unitsForSale = "3";
+          // a unit sale price with a precision up to 12 decimals seems to work in combination with *any* fee (for tokens with 18 decimals).
+          // if *no* fees are involved, 18 decimals may work fine. in-between your mileage may vary, depending on the value of `basisPoints`.
+          const pricePerUnit = "3.1415926535897";
+          const basisPoints = 247;
+
+          // maker creates partially fillable listing with amount of 3
+          standardCreateOrderInput.offer = [
+            {
+              itemType: ItemType.ERC1155,
+              token: testErc1155.address,
+              amount: unitsForSale,
+              identifier: nftId,
+            },
+          ];
+
+          // calculate total price (price per unit * units for sale) and fees
+          standardCreateOrderInput.consideration = [
+            {
+              amount: parseEther(pricePerUnit).mul(unitsForSale).toString(),
```

Comment:
found a couple of other smaller issues, e.g. here we'd have to use `parseUnits(pricePerUnit, 6)` instead to get the correct price

Reply:
oops this comments actually refer to #349ff